src/shared/components/CountdownTimer/api/constants/times.ts

```diff
@@ -0,0 +1,9 @@
+export enum Times {
```

Comment:
you don’t need to keep this enum into api folder. Just in CountdownTimer

Reply:
done